examples/jsm/misc/GroundProjectedEnv.js

```diff
@@ -0,0 +1,160 @@
+import {
+	Texture,
```

Comment:
`Texture` import is unused.

Reply:
Was used in JSDoc comments, removed the comments and the unused import